src/ar/RewayatClub.lua

```diff
@@ -0,0 +1,128 @@
+-- {"id":3302,"ver":"1.0.0","libVer":"1.0.0","author":"Ali Mohamed"}
+
+local baseURL = "https://rewayat.club"
+local baseUrlApi = "https://api.rewayat.club"
+
+
+---@type dkjson
+local json = Require("dkjson")
+
+local function flatten(arr)
+    local results = {}
+    -- define recursive local function for flattening
+    local function arrFlatten(arr)
+        for i, v in ipairs(arr) do
+            if type(v) == "table" then
+                arrFlatten(v)
+            else
+                results[#results+1] = v
+            end
+        end
+    end
+    -- call this functions
+    arrFlatten(arr)
+    return results
+end
+
+return {
+    id = 3302,
+    name = "RewayatClub - نادي الروايات",
+    baseURL = baseURL,
+    hasSearch = true,
+    listings = {
+        Listing("Novel List",true,function (data)
+			local res = Request(GET(baseURL .. "/api/novels/?page=" .. data[PAGE]+1)):body():string()
+
+            local d = json.decode(res)
```

Comment:
```suggestion
            local d = json.GET(baseURL .. "/api/novels/?page=" .. data[PAGE]+1)
```

Reply:
This also applies to other places in the file

Reply:
ok I tried json.Get() method and it give me nil error and tried Request() method and it worked again